packages/core/src/components/search/Filter/FilterSlider.tsx

```diff
@@ -166,13 +166,32 @@ function FilterSlider({
         }
       : undefined
 
+  const allDeliveryMethodsFacetValue = {
```

Comment:
If you agree with the suggestion.

```suggestion
  const allDeliveryMethodsFacet = {
```